regex.mod/regex.bmx

```diff
@@ -238,8 +242,8 @@ Type TRegEx
 			sizeOffsets = pcre2_get_ovector_count_16(matchPtr)
 			offsets = pcre2_get_ovector_pointer_16(matchPtr)
 
-			Local replaceStr:String = replaceWith 
-			Local ofs:Int Ptr = offsets
+			Local replaceStr:String = replaceWith
+			Local ofs:Size_T Ptr = offsets
```

Comment:
you should then possibly make "offsets" also be of type size_t as it is for now defined as
```
	' pointer to the offsets vector, owned by pcre2
?ptr64
	Field offsets:Long Ptr
?Not ptr64
	Field offsets:Int Ptr
?
```

and in line 219 or so
```
	Method Replace:String(target:String, replaceWith:String, startPos:Int = 0)
...
			offsets = pcre2_get_ovector_pointer_16(matchPtr)

			Local replaceStr:String = replaceWith 
			Local ofs:Int Ptr = offsets
```
ofs is an int ptr while it would be size_t then too